src/finalizeEquation.js

```diff
@@ -0,0 +1,17 @@
+import cleanupEquation from "./cleanupEquation";
+import repeatLastEquation from "./repeatLastEquation";
+import runEquation from "./runEquation";
+
+const finalizeEquation = (result, lastEquation) => {
```

Comment:
@grimxreaper great job extracting this! I wonder about the name though? Finalize, while feeling like it covers the scope of the code, doesn't feel like it conveys the most important job of this function. What might be a better name?

Reply:
It changes the equation, but I wasn't sure that 'alterEquation' would be a good name. I'm guessing the most important job of this function is that it returns the correct equation...

Reply:
@grimxreaper Think of this function as its own npm package. What would you call it? When would you tell someone to use it? What problem would they have that this solves?

Reply:
I would call it updateEquation or editEquation...?
I would tell someone to use it when they want to ensure that the equation in the display has been polished and edited to reflect the correct equation. Whether that edit requires the equation to be repeated, ran, or cleaned up.

Reply:
@grimxreaper If I gave someone that function I would tell them it runs the equation! Cleaning it up is just so you can run it.

Reply:
Ah okay yes! I was hesitating to call it that because we already have a runEquation function which also definitely runs the equation. Do you suggest renaming that one?

Reply:
I renamed it, I think it makes more sense now!